test/components/tree-view.spec.tsx

```diff
@@ -47,6 +48,45 @@ const treeData: TreeItemData[] = [
     }
 ];
 
+const newTreeData = JSON.parse(JSON.stringify(treeData));
```

Comment:
I assume that this is meant to copy treeData. Use newTreeData = {...treeData}, it's much clearer.

Reply:
that would be shallow copying

Reply:
Well what do you know
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Deep_Clone

Reply:
please add a comment explaining this

Reply:
also please rename this constant to be clear that it has no real relation to `treeData`